Test.groovy

```diff
@@ -0,0 +1,20 @@
+pipeline {
+  agent any
+  stages {
+    stage('GraphQL engine tests') {
+      steps {
+        sh "docker-compose -f docker-compose-tests.yaml up --abort-on-container-exit postgres graphql-engine hasura-service-tests"
```

Comment:
Here I specify all the services involved in the tests. This is currently not strictly necessary, but in the future it will. Indeed, we currently only have the hasura service. Later on, we will have other services that we want to unit test, without waking up the hasura service.

Reply:
No, it's specific to jenkins.